subprojects/internal-integ-testing/src/main/groovy/org/gradle/integtests/fixtures/executer/OutputScrapingExecutionResult.java

```diff
@@ -391,7 +390,7 @@ private void failOnMissingElement(String message, String expected, Set<String> a
     }
 
     private void failOnMissingOutput(String message, String type, String expected, String actual) {
-        throw new ComparisonFailure(unexpectedOutputMessage(String.format("%s%nExpected: %s%n%n%s:%n=======%n%s", message, expected, type, actual)), expected, actual);
+        throw new AssertionError(String.format("%s%nExpected: %s%n%n%s:%n=======%n%s", message, expected, type, actual));
```

Comment:
What's a good integration test I can use to see the effect of this change? It seems like the "expected" is almost always a single line?